Examples/OneSignalDemo/app/src/main/AndroidManifest.xml

```diff
@@ -1,13 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools"
+    xmlns:amazon="http://schemas.amazon.com/apk/res/android"
```

Comment:
shouldn't all this configuration go under the OneSignal SDK manifest?

Reply:
@Jeasmine No since we don't want to add Amazon by default for everyone.

We could in the future however make a new module to do this for them though

Reply:
a new module sounds good!